src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -148,7 +152,7 @@ export class GlobalMap extends React.Component<Props, State> {
     singleShow: {
       iconImage: Mapbox.StyleSheet.identity("icon"),
       iconSize: 0.8,
-      iconOffset: [0, -21],
+      // iconOffset: [0, -21], @TODO: This property causes with the selected shows icon. Does this need to be here?
```

Comment:
We noticed the pins were off a little bit (not actually pointing to the right location) that's why this was added. we might have to offset the PointAnnotations by that value as well. Since it's a constant it shouldn't be too hard